tagger.py.in

```diff
@@ -15,5 +15,29 @@ else:
 if sys.platform == 'win32':
     os.environ['PATH'] = basedir + ';' + os.environ['PATH']
 
-from picard.tagger import main
-main(os.path.join(basedir, 'locale'), %(autoupdate)s)
+try:
+    from picard.tagger import main
+    main(os.path.join(basedir, 'locale'), %(autoupdate)s)
+except SystemExit:
+    raise  # Just continue with a normal application exit
+except:  # noqa: F722
+    # First try to get traceback information and write it to a log file
+    # with minimum chance to fail.
+    import tempfile
+    import traceback
+    trace = traceback.format_exc()
+    logfile = None
+    try:
+        (fd, logfile) = tempfile.mkstemp(".log", "picard-crash-")
+        os.write(fd, trace.encode(errors="replace"))
+        os.close(fd)
+    except:  # noqa: F722
+        print("Failed writing log file {0}".format(logfile), file=sys.stderr)
+
+    # Display the crash information to the user as a dialog. This requires#
+    # importing Qt5 and has some potential to fail if things are broken.
+    from PyQt5 import QtWidgets
+    app = QtWidgets.QApplication(sys.argv)
```

Comment:
`app` isn't used anywhere

Reply:
Yes, but the variable is needed to keep the object alive. Just using `QtWidgets.QApplication(sys.argv)` without assignment will not work and crash the application.

Reply:
In this case, may be name it `unused` or the like to make clear it is intentional.